Sources/IonicPortals/Portal.swift

```diff
@@ -33,20 +36,23 @@ public struct Portal {
     ///   - name: The name of the portal, must be unique.
     ///   - startDir: The starting directory of the ``Portal`` relative to the root of ``bundle``.
     ///     If `nil`, the portal name is used as the starting directory. Defaults to `nil`.
+    ///   - index: The initial file to load in the Portal. Defaults to `index.html`.
     ///   - bundle: The `Bundle` that contains the web application. Defaults to `Bundle.main`.
     ///   - initialContext: Any initial state rqeuired by the web application. Defaults to `[:]`.
     ///   - liveUpdateManager: The `LiveUpdateManager` responsible for locating the source source for the web application. Defaults to `LiveUpdateManager.shared`.
     ///   - liveUpdateConfig: The `LiveUpdate` configuration used to determine to location of updated application assets. Defaults to `nil`.
     public init(
         name: String,
         startDir: String? = nil,
+        index: String = "index.html",
```

Comment:
Index works for me. The only other suggestion I have is maybe `defaultPage` because that seems to be what the web refers to when talking about the default `index.html` - but I don't have a strong opinion.